cinema/urls.py

```diff
@@ -1,10 +1,63 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    MovieViewSet,
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet
+)
+
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
```

Comment:
The `router.register` for `MovieViewSet` is defined, but individual actions for `movie_list` and `movie_detail` are also specified. This redundancy can lead to confusion. Consider using either the router or individual actions consistently.